app/src/main/java/com/kickstarter/KSApplication.java

```diff
@@ -86,7 +85,7 @@ public boolean isInUnitTests() {
   }
 
   private void checkForCrashes() {
-    final String appId = this.build.isExternal()
+    final String appId = Build.isExternal()
```

Comment:
sorry just saw this... i would think we want this properly injected tho right? why did injection fail?